test/utils/config.ts

```diff
@@ -1,31 +1,34 @@
-import nock from 'nock'
-const fs = require('fs')
-const path = require('path')
+import nock from "nock";
+import fs from "fs";
+import path from "path";
+import { StatusCodes } from "http-status-codes";
 
-export function setConfigToBasic(configId: string): void {
-    const rawContent = fs.readFileSync(path.join(__dirname, '..', 'fixtures', 'config', `${configId}.yml`))
-    const contentBuf = Buffer.from(rawContent)
-    const encodedContent = contentBuf.toString('base64')
-    nock('https://api.github.com')
-        .get('/repos/tianhaoz95/.github/contents/.github/approveman.yml')
-        .reply(200, {
-            type: 'file',
-            encoding: 'base64',
-            size: encodedContent.length,
-            name: 'approveman.yml',
-            path: '.github/contents/.github/approveman.yml',
-            content: encodedContent
-        })
-    nock('https://api.github.com')
-        .get('/repos/tianhaoz95/approveman-test/contents/.github/approveman.yml')
-        .reply(404)
-}
+export const setConfigToBasic = (configId: string): void => {
+  const rawContent = fs.readFileSync(
+    path.join(__dirname, "..", "fixtures", "config", `${configId}.yml`),
+  );
+  const contentBuf = Buffer.from(rawContent);
+  const encodedContent = contentBuf.toString("base64");
+  nock("https://api.github.com")
+    .get("/repos/tianhaoz95/.github/contents/.github/approveman.yml")
+    .reply(StatusCodes.OK, {
+      type: "file",
+      encoding: "base64",
```

Comment:
Codacy found an issue: [The key 'encoding' is not sorted alphabetically](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6083797)

Reply:
done